Paths/SlantRounds.py

```diff
@@ -12,26 +12,28 @@
 thisFont = Glyphs.font # frontmost font
 selectedLayers = thisFont.selectedLayers # active layers of selected glyphs
 
-yShift = []
-xShift = []
 def process( thisLayer ):
-	thisMasterAngle = thisLayer.glyphMetrics()[5]
+	thisMasterAngle = thisLayer.associatedFontMaster().italicAngle
 	myY = thisMasterAngle / 100 
 	myX = -thisMasterAngle / 100 
-	yShift.append(thisLayer.bounds.origin.y)
-	xShift.append(thisLayer.bounds.origin.x)
+	
 	thisLayer.beginChanges()
+		
+	# shear:
+	oldPos = thisLayer.bounds.origin
 	thisLayer.applyTransform([1, myX, myY*2, 1, 0, 0])
 	thisLayer.addNodesAtExtremes()
-	yShift.append(thisLayer.bounds.origin.y)
-	xShift.append(thisLayer.bounds.origin.x)
+	
+	# move back:
+	newPos = thisLayer.bounds.origin
+	xShiftBack = oldPos.x-newPos.x
+	yShiftBack = oldPos.y-newPos.y
+	thisLayer.applyTransform([1, 0, 0, 1, xShiftBack, yShiftBack])
+	
 	thisLayer.endChanges()
+	
+	print "y-shift: %s, x-shift: %s" % (abs(yShift[2]), xShift[2])
```

Comment:
did you mean this?
`print "y-shift: %s, x-shift: %s" % (abs(yShiftBack), xShiftBack)`

Reply:
Yeah, probably. 👍